com.avaloq.tools.ddk.xtext.ide/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,13 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: com.avaloq.tools.ddk.xtext.ide
+Bundle-SymbolicName: com.avaloq.tools.ddk.xtext.ide;singleton:=true
+Bundle-Version: 8.3.0.qualifier
+Bundle-Vendor: Avaloq Evolution AG
+Require-Bundle: org.eclipse.xtext.ui,
```

Comment:
Do we really need a dependency on org.eclipse.xtext.ui? I though the IDE plug-ins are independent on the UI plugins, and if anything the UI plugins are the ones that depend on the IDE ones.

Reply:
Right. Should be org.eclipse.xtext.ide, probably.